CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing Guidelines
+
+* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).  Each active repo has its own stream to direct questions to (for example #powerup or #portal).  Mentorship System stream is [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system).
```

Comment:
[anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio) not [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)